compiler/rustc_mir_transform/src/coverage/tests.rs

```diff
@@ -134,7 +134,7 @@ impl<'tcx> MockBlocks<'tcx> {
             some_from_block,
             TerminatorKind::Call {
                 func: Operand::Copy(self.dummy_place.clone()),
-                args: vec![],
+                args: [].into(),
```

Comment:
`Box<[T]>` implements `Default`, might be easier to use that

Reply:
or `Box::new([])` and let `CoerceUnsized` do the work

Reply:
Well I'd rather you just use `Box::new([x, y, z])` everywhere -- I think trailing `.into()` is just ugly to read and not as analogous to `vec![x, y, z]` 😆

Reply:
Also note that this particular code is just building fake MIR for unit tests, so it's relatively unimportant.

Reply:
Well this is one of many examples -- I wasn't going to leave a comment on all of them, so the fact it's building fake MIR or processing real MIR isn't really relevant